src/session_handle.cpp

```diff
@@ -92,12 +93,12 @@ namespace libtorrent {
 	{
 		std::shared_ptr<session_impl> s = m_impl.lock();
 		if (!s) aux::throw_ex<system_error>(errors::invalid_session_handle);
-		dispatch(s->get_context(), [=]() mutable
+		dispatch(s->get_context(), std::bind([s, f](auto&&... args) mutable
```

Comment:
It would be beneficial to move function `f` into the lambda too if possible.

Reply:
> It would be beneficial to move function `f` into the lambda too if possible.

Generally, yes. But in this case, it is intended only for pointer to member function, so moving it will not add any advantages.

Reply:
I see. So it is a pointer rather than a function object...